### PR TITLE
Progress page polish

### PR DESCRIPTION
**Pull request by rachelchenrc**
Visual polish for Progress/History Page

Screen Shot 2020-01-10 at 2 04 38 PM

[Trello card](https://trello.com/c/2oA8A0xw/70-placeholder-my-history-page-january-polish)
[Live env](https://nj-career-network-dev4.firebaseapp.com/progress/)

### CODE REVIEW

**Comment by rachelchenrc** [comment]
- Marked boxes in the left rail:
In discussion with Alex, we decided to keep using the check for the filter bar, but change the icon in the dashboard to circle. Details was in https://trello.com/c/eZEIpsFY/28-15-allow-jobseekers-to-filter-the-list-view-of-all-activities-history-page-by-activity-category-bucket

- Boarder in the cards:
Currently, the cards in the dashboard are also border-less, we’re going to address it in the following polish story for consistency.

- Divider line above "I felt”: Addressed. The change was in this PR.

- Activity category is not displaying for self-reported activities: this is also a design decision.

File: components/history/CompletedTask.js [hunk 1]
```diff
@@ -62,22 +64,27 @@ function CompleteTask(props) {
...
 
-        <Typography variant="h6" component="h2" className={classes.group}>
+        <Typography component="h1" variant="h5" display="block" className={classes.group}>
           {title}
         </Typography>
-
-        <Typography variant="body2" component="p" className={classes.description}>
-          <Typography component="span" style={{ fontWeight: 'bold' }}>
-            Why?{' '}
+        <div data-intercom="task-why" className={classes.textBlock}>
```
Comment (rossdakin) [inline components/history/CompletedTask.js]: Do we need the `data-intercom` attr here? I don't think we currently have a product tour on this page.

Comment (rachelchenrc) [reply components/history/CompletedTask.js]: removed.

File: components/history/Activity.js [hunk 2]
```diff
@@ -59,46 +65,61 @@ function Activity(props) {
...
       <CardContent>
         <div className={classes.header}>
-          <Typography variant="body1" noWrap style={{ maxWidth: '75%', fontWeight: 500 }}>
+          <Typography variant="body2" noWrap style={{ maxWidth: '75%' }}>
             {activityTypeLabel}
           </Typography>
         </div>
 
-        <Typography variant="h6" component="h2" className={classes.group}>
+        <Typography component="h1" variant="h5" display="block" className={classes.group}>
```
[the remark this answers is not in the document]
Comment (rachelchenrc) [reply components/history/Activity.js]: removed all instances.

File: components/history/Activity.js [hunk 3]
```diff
@@ -110,9 +131,9 @@ function Activity(props) {
         {!!whyIFeelThisWay.length && (
           <div className={classes.group}>
             <Typography variant="body2" className={classes.label} gutterBottom>
-              I felt like this because...
+              I Felt Like This Because...
             </Typography>
-            <Typography variant="body2" component="p">
+            <Typography variant="body2" component="p" color="#2c2f41">
```
[the remark this answers is not in the document]
Comment (rachelchenrc) [reply components/history/Activity.js]: yes, it's quite close, remove it. Sorry for the console error.

**Pull request merged**